### DISCUSSION TITLE
transcluding a function

### DISCUSSION THREAD

**Discussion by dmgerman** [nobiot/org-transclusion#258] [Ideas]
Hi Nobiot,

I finished cleaning up my code. Is this something that you think should be integrated or should it live on its own project?

https://github.com/dmgerman/org-transclude-fn

**Comment by nobiot** [comment]
Great idea. I think it will be a great extension living in our repo here. I have played around with it and have some questions / suggestions. Let me come back to this thread on these. I am on vacation in a remote place :)
